astronomer/providers/core/triggers/external_task.py

```diff
@@ -169,21 +170,21 @@ def serialize(self) -> Tuple[str, Dict[str, Any]]:
 
     async def run(self) -> AsyncIterator["TriggerEvent"]:
         """
-        Makes a series of asynchronous http calls via an http hook poll for state of the job
+        Makes a series of http calls via an http hook poll for state of the job
         run until it reaches a failure state or success state. It yields a Trigger if response state is successful.
         """
         from airflow.utils.state import State
 
-        hook = self._get_async_hook()
+        hook = HttpHook(method="GET", http_conn_id=self.http_conn_id)
```

Comment:
If we're not running it in an async way, is there any reason we need to defer it to the trigger?

Reply:
to free worker slot, no?

Reply:
Yep, but one of the points of this change is that ` A simple request can get the results. `. I thought that it indicates the load would be super light and why can't we use the HTTP provider instad

Reply:
Leaving it to worker will cause a lot of risk to reliability. Workers may get killed for multiple reasons I guess.

Reply:
@Lee-W I think we can use the Http provider with some little extra work. But the idea is to leverage the ExternalTaskSensor.

Reply:
yes, but the trigger can also get killed. and if we use sync API we might get "Triggerer's async thread was blocked for %.2f seconds" warning log in trigger

Reply:
Will that cause an issue? I think I saw that log popping up in the triggerer, even though the task was running just fine.

Reply:
I have created an issue to investigate aiohttp https://github.com/astronomer/astronomer-providers/issues/1308